### PR TITLE
Update file-plan-manager.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed reviewer email

### CODE REVIEW

**Comment by msbemba** [comment]
Issue #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1274

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/file-plan-manager.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ To import new retention labels and modify existing retention labels:
...
    |RetentionType|String|This property specifies whether the retention duration is calculated from the content creation date, event date, labeled (tagged) date, or last modified date. Valid values are:</br>**CreationAgeInDays**</br>**EventAgeInDays**</br>**TaggedAgeInDays**</br>**ModificationAgeInDays** |
-   |ReviewerEmail|SmtpAddress|When this property is populated, a disposition review will be triggered when the retention duration expires. This property specifies the email address of a reviewer for **Delete** and **KeepAndDelete** retention actions. You can include the email address of individual users, distribution or security groups, or Office 365 groups. You can specify multiple email addresses separated by commas.|
+   |ReviewerEmail|SmtpAddress|When this property is populated, a disposition review will be triggered when the retention duration expires. This property specifies the email address of a reviewer for **Delete** and **KeepAndDelete** retention actions. You can include the email address of individual users, distribution or security groups. You can specify multiple email addresses separated by semi Colon.|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/file-plan-manager.md]:
```suggestion
   |ReviewerEmail|SmtpAddress|When this property is populated, a disposition review will be triggered when the retention duration expires. This property specifies the email address of a reviewer for **Delete** and **KeepAndDelete** retention actions. You can include the email address of individual users, distribution, or security groups. You can specify multiple email addresses separated by semicolons.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@cabailey - PR has been copy edited and is ready for final review. This is based on issue #1274.

Thank you.

cc: @kenwith

**Comment by cabailey** [comment]
Reasigning to @chvukosw

**Review by cabailey — COMMENTED** [review]
Assigned to @chvukosw, who is already looking into this

**Pull request merged**